src/components/Registration/Registration.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react'
+import { toast } from 'react-toastify';
+import { Link } from 'react-router-dom';
+
+import AuthService from 'services/AuthService';
+import { validateRegisterData } from 'utils/Validations';
+import { EMAIL_UNAVAILABLE } from 'constants/ErrorMessages';
+
+import './../common_style/Form.scss'
+import './Registration.scss'
+
+const Registration = () => {
+
+	const [user, setUser] = useState({ firstName: '', lastName: '', email: '', password: '' })
+	const [errors, setErrors] = useState({ firstName: '', lastName: '', email: '', password: '', isError: false })
+
+	const handleChange = (e: any) => {
+		setUser(Object.assign({}, user, { [e.target.name]: e.target.value }))
+	}
+
+	const handleSubmit = (e: any) => {
+		e.preventDefault();
+
+		if (validate()) {
+			AuthService.register(
+				user.firstName,
+				user.lastName,
+				user.email,
+				user.password
+			).then(
+				response => {
+					toast.success(JSON.stringify(response.data), { hideProgressBar: true });
+					window.location.replace("/login")
+				}
+			)
+				.catch(() => {
+					toast.error("User registration unsuccessful!", { hideProgressBar: true });
+					setErrors({
+						firstName: '',
+						lastName: '',
+						email: EMAIL_UNAVAILABLE,
+						password: '',
+						isError: true
+					})
+				});
```

Comment:
Theoretically, the entered email address being unavailable is not the only reason why this request might fail. This is fine for now as far as I'm concerned, but think about other reasons why a request might fail and whether it would be confusing for the user to get such an error message under those circumstances.